docker-compose.yml

```diff
@@ -19,6 +21,7 @@ services:
     env_file:
       - './api/.env'
```

Comment:
We need to make two environment files that this base compose file will read from, one for dev and one for prod.  I can add this in the original branch.